### PR TITLE
[332] Fix route definitions / helpers

### LINKED ISSUE #332: Fix all incorrectly-defined routes / routing helpers
Steps to replicate:

1. Visit open group as a student in the same draw
2. Click Request to Join button

Expected: request created
Actual: Routing error (same double param issue as #312 - `draws/3/groups/5/5/request`)

### PR DESCRIPTION
**Pull request by orenyk**
Resolves #332
- Update all custom route definitions to either use `member do` or `collection do` helpers
- Update all links as necessary
- Rename some group actions to make more sense
- Fix add user link in admin nav

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -4,63 +4,63 @@
...
+
+    collection do
+      get 'build'
+    end
   end
+
   resources :enrollments, only: %i(new create)
 
-  resources :draws do
+  resources :draws do # rubocop:disable BlockLength
```
Comment (esoterik) [inline config/routes.rb]: let's just disable this metric for the whole file

**Comment by esoterik** [comment]
basically ready to rebase and merge

**Comment by orenyk** [comment]
rebased and fixed, will merge after green build.

**Comment by orenyk** [comment]
@esoterik approved via Slack

**Pull request merged**